Gemfile.lock

```diff
@@ -24,6 +24,11 @@ GEM
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.3)
+    active_model_serializers (0.10.7)
```

Comment:
We talked about not having to use serializers... I would only use them when I had really custom JSON that I wanted back. Otherwise, I would not use serializers in real life.